### ISSUE TITLE
No such file or directory: 'Gen.pt' in tsne.py

### ISSUE THREAD

**Issue by stillwang96** [Kadam-Tushar/ajrnn-pytorch#2]
Hello, what is the Gen.pt in tsne.py?

**Comment by Kadam-Tushar** [comment]
@stillwang96
`Gen.pt` contains saved model weights:
 
The generator part of aj-rnn model trained on the dataset mentioned in the paper. 

The `tsne.py` file is helpful for finding low dimensional embeddings (2 dim) of hidden states generated by the model for some examples.
The aim is to find out how the effectiveness model's hidden states by analyzing visual separability in  2 dimensions.  
 
 Let me try to find if I have gen.pt in my storage but you can always re-run the code and get your gen.pt .

**Comment by stillwang96** [comment]
First of all, thank you very much for your reply to my question. I re-run the code, but I didn't find where to save the Gen.pt in the code and what to save in it. Could you help me to finish the saving part in the code? Thanks again for your help!
